measurement/plugins/download_speed/measurements.py

```diff
@@ -219,16 +219,19 @@ def _get_wget_results(self, url, download_timeout):
             return self._get_wget_error("wget-regex", url, traceback=wget_out.stderr)
 
         try:
-            storage_unit = NetworkUnit(
-                match_data.get("download_unit").replace("MB/s", "Mbit/s")
-            )
-        except ValueError:
+            download_rate_unit_map = {
```

Comment:
The dictionary declaration should not be in the try / except as it masks where the error is coming from. It could be set on the class / module level to make it only create the variable allocation at run time rather than at each iteration

Reply:
Good point. Fixed.